### PR TITLE
cloud-games: add loading indicator, do not request data in parallel

### PR DESCRIPTION
**Pull request by WorldLanguages**
1. Adds loading indicator
2. Shows incomplete data after more than 50% of projects loaded, but keeps the loading indicator until 100% loads
3. Instead of starting each request in parallel, request every 125ms

### CODE REVIEW

**Comment by RedGuy12** [comment]
why not in parallel?
________________________________

File: popups/cloud-games/popup.html [hunk 1]
```diff
@@ -10,21 +10,31 @@
...
-        <a class="project-name" :href="'https://scratch.mit.edu/projects/' + project.id" target="_blank"
-          >{{ project.title }}</a
-        >
-        <span class="float-right"><img class="small-icon" src="../../images/icons/users.svg" /> {{ project.amt }}</span>
-      </div>
-      <div class="project-details" v-show="true || project.amt > 0 && project.extended">
-        <div class="username-list">
-          <a v-for="user of project.users" :href="'https://scratch.mit.edu/users/' + user" target="_blank"
-            >{{ user }}</a
+  <body style="display: none" v-show="true">
```
Comment (TheColaber) [inline popups/cloud-games/popup.html:13]: not sure why you need those 2 atterabutes

File: popups/cloud-games/popup.html [hunk 2]
```diff
@@ -10,21 +10,31 @@
...
+    <div
+      v-show="projects.length === 0 || projects.length !== projectsChecked"
+      style="text-align: center; font-size: 0.9rem; padding-top: 1rem; font-weight: bold"
```
Comment (TheColaber) [inline popups/cloud-games/popup.html:16]: could we have this style in the CSS file plz

**Review by TheColaber — CHANGES_REQUESTED** [review]

File: popups/cloud-games/popup.html (hunk 1 above)
Comment (WorldLanguages) [reply popups/cloud-games/popup.html:13]: hide the body until vue loads

**Comment by WorldLanguages** [comment]
> why not in parallel?

not a scientific explanation, but feels wrong to start more than 10 requests at the same time. scratch often suggests doing less than 10 req/s

**Pull request merged**